### PR TITLE
fix upgrade dependencies

### PR DESCRIPTION
**Pull request by Zacharysp**
Addresses issue #66 

- upgrade loopback to 3.26.0
- upgrade jest to 24.8.0
- upgrade commitlint/cli to 8.1.0
- upgrade commitlint/config-conventional to 7.6.0

There are still 3 moderate severity vulnerabilities related to swagger-ui that affected loopback-component-explorer. 

loopback-component-explorer use swagger-ui@2 which is deprecated, and they think it's to much effort to make the upgrade to swagger-ui@3., here is the ref issue:
https://github.com/strongloop/loopback-component-explorer/issues/263#issue-458983127

### CODE REVIEW

**Comment by makara** [comment]
- @xavierchow's change was merged. Please rebase
- Try `yarn upgrade-interactive --latest` and update all the green and yellow ones
- See if you could upgrade `loopback-boot` from 2 to 3

**Comment by Zacharysp** [comment]
- upgrade bluebird to 3.5.5
- upgrade helmet to 3.20.0
- upgrade loopback to 3.26.0
- upgrade loopback-boot to 3.3.1
- upgrade loopback-connector to 4.8.0
- upgrade strong-remoting to 3.14.0

- upgrade commitlint/cli to 8.1.0
- upgrade commitlint/config-conventional to 7.6.0
- upgrade coveralls to 3.0.6
- upgrade eslint-plugin-prettier to 3.1.0
- upgrade husky to 3.0.3
- upgrade jest to 24.8.0
- upgrade nodemon to 1.19.1
- upgrade prettier to 1.18.2

**Comment by haishanh** [comment]
Might be a good idea to have a [Dependabot](https://github.com/marketplace/dependabot-preview) here, it's quite nice.

(Dependabot is a deps mgmt bot, simliar to Green Keeper, it's acquire by GitHub, now free to use for both public and private repos.)

File: test/e2e/testAPI.test.js [hunk 1]
```diff
@@ -4,15 +4,8 @@ const request = require('supertest');
...
-  it('should return 204 response', () => {
-    request(app)
-      .get('/api/test')
-      .set('Accept', 'application/json')
-      .expect(204);
-  });
-
   it('should return 404 response', () => {
-    request(app)
+    return request(app)
```
Comment (xavierchow) [inline test/e2e/testAPI.test.js]: good catch

**Comment by xavierchow** [comment]
@Zacharysp could you merge the PR?

**Comment by Zacharysp** [comment]
Seems like I don't have permission to merge.

**Comment by xavierchow** [comment]
What if we remove `loopback-component-explorer`?
I barely use it coz I always have my test code to access the APIs.

**Pull request merged**